README.md

```diff
@@ -9,13 +7,16 @@ Status for development branch
 [![Build Status](https://travis-ci.org/automl/CAVE.svg?branch=development)](https://travis-ci.org/automl/CAVE)
 
 # CAVE 
-CAVE is an analyzing tool that builds on SMAC3 (https://github.com/automl/SMAC3)
+CAVE is an analyzing tool for automatic algorithm configurators.
+The current version works out-of-the-box with [SMAC3](https://github.com/automl/SMAC3), but can be easily adapted to other configurators, as long as they use the same output-structure.
 
 # LICENSE 
-Please refer to LICENSE (https://github.com/automl/CAVE/blob/master/LICENSE)
+Please refer to [LICENSE](https://github.com/automl/CAVE/blob/master/LICENSE)
 
 # OVERVIEW 
-CAVE is an analyzing tool. It is written in Python 3.6 and uses SMAC3. CAVE generates performance-values (e.g. PAR10), scatter- and cdf-plots to compare the default and the optimized incumbent and providing further inside into the optimization process by quantifying the parameter importance.
+CAVE is an analyzing tool.
+It is written in Python 3.6 and uses [SMAC3](https://github.com/automl/SMAC3), [ParameterImportance](https://github.com/automl/ParameterImportance) and [ConfigSpace](https://github.com/automl/ConfigSpace).
```

Comment:
Later python 3.5 is required, but we write that it is written in 3.6 (which implies that it needs 3.6)
Also I'd writhe [pimp] instead of [ParameterImportance]